graviti/dataframe/column/series.py

```diff
@@ -241,6 +244,11 @@ def _extend(self: _SB, values: _SB) -> None:
             5  3
 
         """
+        if self._parent is not None:
```

Comment:
we should put all checks in public `extend`, not in private `_extend`

Reply:
Series does not support `extend` now. So we do not need to add this check because using private methods is not recommended.